src/Altinn.Platform/Altinn.Platform.Events/Tests/Altinn.Platform.Events.Tests.csproj

```diff
@@ -16,14 +17,8 @@
     <PackageReference Include="coverlet.collector" Version="1.3.0" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(Configuration)'=='Debug'">
-    <PackageReference Include="StyleCop.Analyzers" Version="1.1.118">
-      <PrivateAssets>all</PrivateAssets>
-      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
-    </PackageReference>
-    <AdditionalFiles Include="..\..\..\..\stylecop.json">
-      <Link>stylecop.json</Link>
-    </AdditionalFiles>
```

Comment:
We want to use StyleCop. It helps us get early warning about code issues instead of waiting for a SonarCloud report.

Reply:
Done :)